backend/private-graph/graph/schema.resolvers.go

```diff
@@ -7864,6 +7864,24 @@ func (r *sessionResolver) DeviceMemory(ctx context.Context, obj *model.Session)
 	return deviceMemory, nil
 }
 
+// SessionComments is the resolver for the session_comments field.
+func (r *sessionResolver) SessionComments(ctx context.Context, obj *model.Session) ([]*model.SessionComment, error) {
```

Comment:
Resolver to return session comments preloaded with attachments and replies.
Not currently using the attachments and replies on UI at the moment. Have to consider removing the
preload.